package.json

```diff
@@ -38,6 +38,7 @@
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.33.0",
     "babel-eslint": "^10.0.3",
+    "classnames": "^2.5.1",
```

Comment:
classnames is already installed in projects that need it. You don't have to install it, so it should be removed here.

Reply:
Also, as there are no further changes to packages, you should remove the changes you've made to package-lock.json from the PR (but don't delete the file altogether).